conf/nginx/rate-limiting.conf.jnj

```diff
@@ -15,7 +15,7 @@ map $http2 $http2_bucket {
 # Extract namespace from scope query parameter
 # Docker scope format: "repository:namespace/repo:actions"
 map $arg_scope $scope_namespace {
-  ~^repository:([^/]+)/.* $1;
+  ~^repository(?:|%3A)([^(?\/|%2F)]+)[(?\/|%2F)].+ $1;
```

Comment:
looking at this regex, if all you want is to match something like `repository%3Aredhat-user-workloads%2Fsomerepo%3Apull%2Cpush` or `repository%3Asome-repo%3Apull%2Cpush`  you can simplify this to `^repository%3A(.+?)(%2F|%3A).*` what do you think?